content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Frontend Overhaul of the OpenTelemetry Demo (Go to Next.js)
+linkTitle: Frontend Overhaul
+date: 2022-10-18
+author: "[Oscar Reyes](https://github.com/xoscar) (Tracetest)"
+draft: true
+canonical_url: https://tracetest.io/blog/frontend-overhaul-opentelemetry-demo
+---
+
+One of the OpenTelemetry Project's many Special Interest Groups (SIG) is the
+OpenTelemetry Community Demo SIG which gives support to a set of instrumented
+backend microservices and a web frontend app that are primarily used to showcase
+how to instrument a distributed system using OpenTelemetry.
+
+The application's main focus is to demonstrate the implementation process to
+instrument an application no matter what programming language, platform, or
+operating system your team is using, as well as providing different approaching
+techniques (automatic and manual instrumentation, metrics, baggage). All of this
+while following the standards and conventions defined by the official
+OpenTelemetry Documentation. More about the specific requirements can be
+[found here](https://github.com/open-telemetry/opentelemetry-demo/tree/main/docs/requirements).
+
+My company was focused on becoming part of and embracing the OpenTelemetry
+community. One of our goals this summer was to get more involved with a core
+OpenTelemetry project where we could provide a meaningful contribution. The OTel
+demo became the best match for achieving that goal as it would not only help the
+community, but help our product as we needed a good example to test and showcase
+what can be done with our tool.
+
+The first thing we did was to get in contact with Carter Socha, the organizer of
+the OTel Demo SIG. Carter was really welcoming and helped us identify where our
+contributions could be the most impactful. We started looking at
+[the issue created by Austin Parker](https://github.com/open-telemetry/opentelemetry-demo/issues/39)
+referencing a complete front-end overhaul that would involve moving the
+application away from Go (SSR) to an architecture that included a browser-side
```

Comment:
what does SSR, CSR stand for? don't expect people to know that, introduce those abbreviation or link to a definition eg wikipedia